specification/Metadata/README.md

```diff
@@ -389,46 +427,44 @@ For a table with `N` rows, the buffer that stores these boolean values will cons
 
 A string value is a UTF-8 encoded byte sequence. Multiple strings are packed tightly in the same buffer.
 
-Because string lengths may vary, a **string offset** buffer (`stringOffset`) is used to identify strings in the buffer. If there are `N` strings in the property array, the string offset buffer has `N + 1` elements. The first `N` of these point to the first byte of each string, while the last points to the byte immediately after the last string. The number of bytes in the `i-th` string is given by `stringOffset[i + 1] - stringOffset[i]`. UTF-8 encodes each character as 1-4 bytes, so string offsets do not necessarily represent the number of characters in the string.
+Because string lengths may vary, a **string offset** buffer is used to identify strings in the buffer. If there are `N` strings in the property array, the string offset buffer has `N + 1` elements. The first `N` of these point to the first byte of each string, while the last points to the byte immediately after the last string. The number of bytes in the `i-th` string is given by `stringOffset[i + 1] - stringOffset[i]`. UTF-8 encodes each character as 1-4 bytes, so string offsets do not necessarily represent the number of characters in the string.
 
-The data type used for offsets is defined by a **string offset type** (`stringOffsetType`), which may be `UINT8`, `UINT16`, `UINT32`, or `UINT64`.
+The data type used for offsets is defined by a **string offset type**, which may be `UINT8`, `UINT16`, `UINT32`, or `UINT64`.
 
 > **Example:** Three UTF-8 strings, binary-encoded in a buffer.
 >
 > ![String property example](figures/unicode-strings.png)
 
 #### Enums
 
-Enums are encoded as integer values according to the enum value type (see [Enums](#enums)). Multiple enum values are packed tightly in the same buffer. Any integer data type supported for [Numbers](#numbers) may be used for enum values.
+Enums are encoded as integer values according to the enum value type (see [Enums](#enums)). Any integer data type supported for [Scalars](#scalars) may be used for enum values.
 
-#### Arrays
+#### Fixed-Length Arrays
 
-Array values are encoded with varying array lengths and element sizes. Multiple arrays and array values are packed tightly in the same buffer.
+A fixed-length array value is encoded as a tightly packed array of `count` elements, where each element is encoded according to the `type`.
 
-Variable-length arrays use an additional **array offset** buffer (`arrayOffset`). The `i-th` value in the array offset buffer is an element index — not a byte offset — identifying the beginning of the `i-th` array. String values within an array may have inconsistent lengths, requiring both array offset and **string offset** buffers (see: [Strings](#strings)).
+#### Variable-Length Arrays
```

Comment:
@lilleyse Question, for a type like this:

```js
{
  array: true,
  type: "VEC3",
  componentType: "FLOAT32"
}
```

And data like this:

```
vectorValues: [
  [Cartesian3(1, 1, 1), Cartesian3(2, 2, 2), Cartesian3(3, 3, 3)],
  [Cartesian3(4, 4, 4)]
]

packedValues = [1, 1, 1, 2, 2, 2, 3, 3, 3, 4, 4, 4]
```

Which is correct for the array offsets?:

```
// counting float components (I _think_ this is correct?)
arrayOffsets = [0, 9, 12]
// counting vector components
arrayOffsets = [0, 3, 4]
```

Reply:
It would be the second case, the index of the vector element. I added that as an example in https://github.com/CesiumGS/3d-tiles/pull/630/commits/3ed7328929547a17936cc8760facb617434e1568.